results/server/demo/demo.py.j2

```diff
@@ -0,0 +1,57 @@
+import pytest
+from ra_test import RATest
```

Comment:
do we want to keep this file? if so lets update it with the newer jinja contents

Reply:
We can update it when testing. Kinda just made this to run it without the server as a separate version. I find it helpful but we can remove it

Reply:
I'm fine with keeping the demo folder, just want to have it up to date